src/graphql/committers-query.ts

```diff
@@ -0,0 +1,29 @@
+export default `
+query orgs($login: String!, $since: GitTimestamp!, $until: GitTimestamp!) {
+    organization(login: $login) {
+        repositories(first: 100) {
+            nodes {
+                name
+                defaultBranchRef {
+                    name
+                    target {
+                        ... on Commit {
+                            history(since: $since, until: $until) {
+                                nodes {
+                                    committedDate
```

Comment:
not strictly required to include this attribute in the response but it would become impossible to unit test whether the usernames list result really only include commits between the given range.
We can leave it up to the server/caller to filter/remap the response as needed.